src/detectors/typeChecker/index.ts

```diff
@@ -156,12 +161,16 @@ export class TsProjectDetector extends ProjectBasedDetector {
 		const reader = this.project.getReader({
 			style: "buildtime",
 		});
+		const rootReader = this.project.getRootReader();
 
 		// Read all resources and test-resources and their content since tsc works completely synchronous
 		const globEnd = taskStart("Locating Resources");
 		const fileTypes = "{*.js,*.view.xml,*.fragment.xml,manifest.json}";
 		const allResources = await reader.byGlob("/resources/**/" + fileTypes);
 		const allTestResources = await reader.byGlob("/test-resources/**/" + fileTypes);
+
+		const rootFileTypes = "{ui5.yaml,*-ui5.yaml,*.ui5.yaml,ui5-deploy.yaml,ui5-dist.yaml,ui5-local.yaml}";
```

Comment:
why not checking for `ui5-*.yaml`, so `ui5-deploy.yaml,ui5-dist.yaml,ui5-local.yaml` can be removed

Reply:
Agree. I think we discussed this already. The only reason for having those specific names in the JSON schema catalog is that `ui5-*.yaml` would also match for `ui5-app/config.yaml`, which is unexpected. We had that initially, but then changed it to some known file names.

Reply:
Yes

Reply:
done